app/models/server/models/LivechatDepartmentAgents.js

```diff
@@ -11,9 +11,9 @@ export class LivechatDepartmentAgents extends Base {
 		super('livechat_department_agents');
 
 		this.tryEnsureIndex({ departmentId: 1 });
-		this.tryEnsureIndex({ departmentEnabled: 1 });
-		this.tryEnsureIndex({ agentId: 1 });
-		this.tryEnsureIndex({ username: 1 });
+		this.tryEnsureIndex({ departmentEnabled: 1, agentId: 1 });
```

Comment:
don't forget the migration 😬

Reply:
😬 i'm scared of index migrations now lol